db/re/item_db_equip.yml

```diff
@@ -43096,11 +43096,12 @@ Body:
     Trade:
       NoDrop: true
     Script: |
+      .@bonus = max(getskilllv("TF_DOUBLE"), 2);
```

Comment:
You don't need to add this, because  it has hardcode inside battle.cpp  (battle_calc_multi_attack).